### ISSUE TITLE
Grandia Saturn (Beetle Core 1.29) FMVs slowdown/fps drop

### ISSUE THREAD

**Issue by Sfree4ever** [libretro/RetroArch#15426]
## Description

Grandia Saturn in Beetle Core FMVs slowdown/fps drop (only FMVs)

### Expected behavior

Considering that beetle core is 1.29 (based on mednafen 1.29)
it should work without any problems and smooth like in mednafen
So it must be something with Retroarch front end

### Actual behavior

When the FMVs kick in the FPS starts to drop and the video stutter/slowdown with crackling in audio

### Steps to reproduce the bug

Just run the game normal

### Bisect Results

always happens

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.15
- Beetle Core: 1.29 

### Environment information

- OS: Windows 10

**Comment by bslenul** [comment]
Tested the original Japanese version and an English translated one, no issue for me. I can reproduce if I enable run-ahead however (Settings > Latency), so you might want to disable it if you're using it.

**Comment by Sfree4ever** [comment]
That's weird because, I tested on 2 different devices with windows 10, 

laptop with Intel(R) Core(TM) i5-10300H CPU @ 2.50GHz  + RTX 2060
Retroarch 1.9.7 + Beetle 1.29

and 

Steam deck with Windows 10 
Retroarch 1.15 + Beetle 1.29

And it happens in both

Both don't have Run-Ahead nor Run Preemptive Frames turned on

**Comment by Sfree4ever** [comment]
I tested it with i7 9700 + gtx 1080ti and in vulkan it still happens, only way I could fix it was turn off vsync in nvidia panel, and vsync on in retroarch, but then screen tearing happen, but the FMVs worked fine this way.

When I switched to gl, the FMVs worked fine in this PC. 
In the laptop and steam deck, I tried both vulkan and gl, and it always happen.
SO the only way to fix it, I think it is to have a powerful PC and run it in gl mode.

**Comment by Alextended** [comment]
Same issue here in the beetle saturn core for all these years since it was fixed in stand alone. No similar issue in any other game I've tested, no issue in stand alone, with all kinds of vsync/video/outputwhatever settings tested in the core and the proposed solutions as here.

**Comment by gouchi** [comment]
> Can't find an active beetle Saturn core github section to actually report it properly.

https://github.com/libretro/beetle-saturn-libretro/issues/35 ?

**Comment by McRmie** [comment]
> Same issue here in the beetle saturn core for all this time since it was fixed in stand alone. No similar issue in other games I've tested, no issue in stand alone, with all kinds of vsync/video/output/whatever settings tested in the core and the proposed solutions.
> 
> Can't find an active beetle Saturn core github section to actually report it properly.

Fixed on my end [here:](https://github.com/libretro/beetle-saturn-libretro/issues/35#issuecomment-2781725766)

**Comment by Vamp898** [comment]
Can reproduce on my end, no workaround helped

**Comment by Alextended** [comment]
Yeah, the issue has persisted for me as well throughout all this time.

Mind I recently got a new (modest, no flagship or anything) phone, the Poco X7 Pro, and it's strong enough for Beetle Saturn and the issue with these FMV does NOT occur there so it's something specific to the Windows build or however these work.
